stock_picking_batch_extended/__manifest__.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2012-2014 Alexandre Fayolle, Camptocamp SA
+# Copyright 2018 Tecnativa - Carlos Dauden
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Stock batch picking",
+    "summary": "Allows manage a lot of pickings in batch",
+    "version": "13.0.1.0.0",
+    "author": "Camptocamp, " "Tecnativa, " "Odoo Community Association (OCA)",
+    "development_status": "Mature",
+    "maintainers": ["Camptocamp", "Tecnativa"],
+    "category": "Warehouse Management",
+    "depends": ["stock_picking_batch", "delivery", "stock_picking_mass_action"],
```

Comment:
Why did this module depends on delivery?

Reply:
Hi!! this module depends on delivery because we print carrier info from picking.
And carrier_id field is created by delivery module.